### PR TITLE
Add mc admin console command

### PR DESCRIPTION
**Pull request by poornas**
This command displays console logs from minio server

Corresponding minio PR: https://github.com/minio/minio/pull/7784

### CODE REVIEW

**Comment by sinhaashish** [comment]
@poornas  Can you check the travis failure. Failing with the below message.
`go: parsing ../minio/go.mod: open /home/travis/gopath/src/github.com/minio/minio/go.mod: no such file or directory`

**Comment by poornas** [comment]
> @poornas Can you check the travis failure. Failing with the below message.
> `go: parsing ../minio/go.mod: open /home/travis/gopath/src/github.com/minio/minio/go.mod: no such file or directory`

for now it will fail as the minio PR is not merged, the go.mod has a replace directive to point to the local minio/minio repo on your file system

File: cmd/admin-log.go [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+	  {{.HelpName}} - {{.Usage}}
+  
+  USAGE:
+	  {{.HelpName}} [FLAGS] TARGET
+  
+  FLAGS:
+	  {{range .VisibleFlags}}{{.}}
+	  {{end}}
+  EXAMPLES:
+		  1. Show console logs for a MinIO server with alias 'play'
```
Comment (krisis) [inline cmd/admin-log.go]: The indentation seems a bit off.

File: cmd/admin-log.go [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+  `,
+}
+
+func checkAdminLogSyntax(ctx *cli.Context) {
+	if len(ctx.Args()) != 1 {
+		cli.ShowCommandHelpAndExit(ctx, "log", 1) // last argument is exit code
+	}
+}
+
+type logMessage struct {
```
Comment (krisis) [inline cmd/admin-log.go]: Could you add a comment explaining the purpose of this struct? Like it is for adding `JSON` and `String` methods etc.

File: go.mod [hunk 3]
```diff
@@ -44,3 +44,5 @@ require (
 )
 
 replace github.com/gorilla/rpc v1.2.0+incompatible => github.com/gorilla/rpc v1.2.0
+
+replace github.com/minio/minio => ../../minio/minio
```
Comment (krisis) [inline go.mod]: Leaving a comment here to remind us to change this line once the minio server PR is merged.

File: cmd/admin-log.go (hunk 1 above)
Comment (poornas) [reply cmd/admin-log.go]: fixed this - appears correctly on terminal

**Comment by poornas** [comment]
@krisis and @sinhaashish , renamed the command to `mc admin console` and pushed an update to allow filtering logs by nodename or  use --limit flag to limit previous logs to display. Server will buffer up to 10k log entries

**Comment by kannappanr** [comment]
Blocking this PR until minio/minio#7784 gets merged

File: cmd/admin-console.go [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+  
+  USAGE:
+	  {{.HelpName}} [FLAGS] TARGET
+  
+  FLAGS:
+	  {{range .VisibleFlags}}{{.}}
+	  {{end}}
+  EXAMPLES:
+		1. Show console logs for a MinIO server with alias 'play'
+			$ {{.HelpName}} play
```
Comment (harshavardhana) [inline cmd/admin-console.go]: These examples should be formatted properly.

**Pull request merged**